terra_layer/views.py

```diff
@@ -54,10 +55,11 @@ def get(self, request, slug=None, format=None):
             raise Http404('View does not exist')
 
         view = settings.TERRA_LAYER_VIEWS[slug]
-        layers = self.layers(view['pk'])
 
-        return Response(
-            {
+        response_dict = cache.get('terra-layer-{}'.format(view['pk']))
```

Comment:
Could you create a method to get the layer's cache key to unify its formating?

Reply:
Then, maybe cache.get_or_set could be used here, don't you think ?

Reply:
Added through a new method

Reply:
:+1: